docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -159,6 +159,8 @@ node.
 <5> The decider which led to the `no` decision for the node.
 <6> An explanation as to why the decider returned a `no` decision, with a helpful hint pointing to the setting that led to the decision. In this example, a newly created index has <<indices-get-settings,an index setting>> that requires that it only be allocated to a node named `nonexistent_node`, which does not exist, so the index is unable to allocate.
 
+See this https://www.youtube.com/watch?v=5z3n2VgusLE[fixing "node does not match index setting" video] for an example troubleshooting walkthrough.
```

Comment:
"example" and "walkthrough" are kind of the same thing. we also don't need to provide the title of the video in the link.

```suggestion
See https://www.youtube.com/watch?v=5z3n2VgusLE[this video] for a walkthrough of troubleshooting a node and index setting mismatch.
```